From: Matthias Klose <doko@debian.org>
Date: Sat, 29 Oct 2022 21:23:31 -0400
Subject: Debian: Suggest installation of python3-gdbm package

We split gdbm out into a separate binary package. Help users who try to import
it, without it installed.
Forwarded: not-needed
---
 Lib/dbm/gnu.py  | 5 ++++-
 Lib/dbm/ndbm.py | 5 ++++-
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/Lib/dbm/gnu.py b/Lib/dbm/gnu.py
index b07a1de..24ba36f 100644
--- a/Lib/dbm/gnu.py
+++ b/Lib/dbm/gnu.py
@@ -1,3 +1,6 @@
 """Provide the _gdbm module as a dbm submodule."""
 
-from _gdbm import *
+try:
+    from _gdbm import *
+except ImportError as msg:
+    raise ImportError(str(msg) + ', please install the python3-gdbm package')
diff --git a/Lib/dbm/ndbm.py b/Lib/dbm/ndbm.py
index 23056a2..44125e7 100644
--- a/Lib/dbm/ndbm.py
+++ b/Lib/dbm/ndbm.py
@@ -1,3 +1,6 @@
 """Provide the _dbm module as a dbm submodule."""
 
-from _dbm import *
+try:
+    from _dbm import *
+except ImportError as msg:
+    raise ImportError(str(msg) + ', please install the python3-gdbm package')
